content/en/golden_signals.md

```diff
@@ -0,0 +1,22 @@
+---
+title: Golden Signals
+status: Feedback Appreciated
+category: concept
+---
+
+## What it is
+Golden signals contain a set of criteria that provide a wide view of a service from a user’s perspective . It is the gold standard for monitoring a web application’s performance.
```

Comment:
The first sentence is a mouth full :) Remember that we are trying to make this accessible to anyone.  

The most basic information we need to convey is that golden signals are metrics. Signal doesn't necessarily imply that. That's the first aha moment. Then we need to explain what kind of metrics. There are typically four: latency, traffic, errors, and saturation. Followed by a quick note as to why they are important. Why should you care about golden signals? Does that make sense?

Golden signals are key metrics for any application, not only web apps, btw.

Reply:
Also likely worth linking out to one of the google SRE docs where they define golden metrics, ex: https://sre.google/sre-book/monitoring-distributed-systems/#xref_monitoring_golden-signals